src/schema/README.md

```diff
@@ -601,10 +637,7 @@ README:
   level: required
   stem: README
   extensions:
-    - ''
-    - .md
-    - .rst
-    - .txt
+    - ['', '.md', '.rst', '.txt']
```

Comment:
This change has not been merged.

Reply:
yeah
so revert this, merge this PR first and amend it in the columbus PR (#1492) ?

Reply:
Yes.